ai/pipelines/langgraph_workflow.py

```
@@ -81,72 +179,231 @@ async def _synthesize_response(calculation: dict, collected_parameters: dict, in
+    if _should_include_legal_basis(collected_parameters, breakdown):
+        try:
+            from ai.tools.law_search.wrapper import search_law_tool
+
+            # 검색 쿼리 생성
+            queries = _generate_law_search_queries(collected_parameters, breakdown)
+            LOGGER.info(f"Searching law for queries: {queries}")
+
+            # 각 쿼리별로 검색 (비동기 처리 가능하지만 단순화를 위해 순차 처리)
+            all_citations = []
+            for query in queries:
+                result = search_law_tool(query, top_k=10)  # 쿼리당 10개로 증가 (3개 쿼리 × 10개 = 최대 30개)
```

Ruff (0.14.0)

Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)